lib/rouge/lexers/python.rb

```diff
@@ -137,7 +137,16 @@ def current_string
           end
         end
 
-        rule identifier, Name
+        # dotted identifiers can still be builtins!
+        rule identifier do |m|
+          if self.class.builtins.include? m[0]
+            token Name::Builtin
+          elsif self.class.builtins_pseudo.include? m[0]
+            token Name::Builtin::Pseudo
+          else
+            token Name
+          end
+        end
```

Comment:
The negative lookbehind to prevent things like `thing.from` being highlighted as keywords... never worked until now. Turns out we do want builtins highlighted after `.`.